src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -516,7 +516,7 @@ std::vector<::TerminalApp::SettingsLoadWarnings> winrt::TerminalApp::implementat
                 warnings.insert(warnings.end(), parseWarnings.begin(), parseWarnings.end());
 
                 // if an arg parser was registered, but failed, bail
-                if (args == nullptr)
+                if (pfn && args == nullptr)

```

Comment:
I’m surprised we are getting in here, though. Don’t we drop out because there isn’t a deserializer registered for null or unbound?

Reply:
Should we just put this _all_ under “if pfn” and move on? After all, there are no warnings if there’s no pfn..

Reply:
there's actually an explicit `nullptr` deserializer registered for "unbound"